picard/options.py

```diff
@@ -171,8 +173,14 @@
 # picard/ui/options/cover_processing.py
 # Cover Art Image Processing
 BoolOption('setting', 'filter_cover_by_size', False)
-IntOption('setting', 'cover_minimum_width', 250)
-IntOption('setting', 'cover_minimum_height', 250)
+IntOption('setting', 'cover_minimum_width', DEFAULT_COVER_MIN_SIZE)
+IntOption('setting', 'cover_minimum_height', DEFAULT_COVER_MIN_SIZE)
+BoolOption('setting', 'resize_images_saved_to_tags', False)
+IntOption('setting', 'cover_tags_maximum_width', DEFAULT_COVER_MAX_SIZE)
+IntOption('setting', 'cover_tags_maximum_height', DEFAULT_COVER_MAX_SIZE)
+BoolOption('setting', 'resize_images_saved_to_file', False)
+IntOption('setting', 'cover_file_maximum_width', DEFAULT_COVER_MAX_SIZE)
+IntOption('setting', 'cover_file_maximum_height', DEFAULT_COVER_MAX_SIZE)
```

Comment:
1. I am unclear why files and tags have separate maximum sizes but not separate minimum sizes?
2. Unclear why `cover_tags_maximum_width` is plural `tags` but `cover_file_maximum_width` is singular `file`?